react/basic-example/src/components/Chatbox.js

```diff
@@ -0,0 +1,39 @@
+import React, { useContext, useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import { TalkSessionContext } from '../contexts/TalkSessionContext';
+
+/* 
+This is an example component that uses one of TalkJS' pre-built UI modes - or more specifically, the 'Chatbox' UI mode
+The Chatbox UI is typically used when the chat may be related to a speicific 'topic' such as a video, product order etc.
+*/
+function Chatbox(props) {
+
+    const chatContainer = useRef();
+    const {me, talkSession} = useContext(TalkSessionContext);
+
+    useEffect(() => {
+
+        if(talkSession && props.conversationId) {
+            
+           const conv = talkSession.getOrCreateConversation(props.conversationId); 
```

Comment:
I'd name this variable `conversation` in full, to be more explicit.

Reply:
Good point, fixed!